src/test/java/ResourceProcessorTests.java

```diff
@@ -86,7 +85,7 @@ public void readListTest() throws Exception {
         mockMvc.perform(get(urlTemplate))
                 .andExpect(status().isOk());
 
-        assertFalse(CitizenResourceProcessor.GOT_CALLED);
+        assertTrue(CitizenResourceProcessor.GOT_CALLED);
```

Comment:
Yes. 

* `Resources` -based processor is tapped to let you set links on a collection resource. 
* `Resource` -based processor is tapped to let you set links on each individual item's resource.

Now I strongly favor using `Resource<Foo>` over `Foo extends ResourceSupport`. Feels like a nice separation of concerns.